src/decycle.ts

```diff
@@ -1,36 +1,38 @@
 import { isPlainObject } from './isPlainObject';
 import { pathToPointer } from './pathToPointer';
 
-export const decycle = (obj: unknown, replacer?: (value: any) => any) => {
+export function decycle(obj: unknown, replacer?: (value: any) => any) {
```

Comment:
How does this optimization help?

Reply:
Its related to all three PR's , not this single file helps to reduce timing.